example.env

```diff
@@ -1,3 +1,4 @@
 REACT_APP_APIKEY=
 REACT_APP_HERE_APIKEY=
-REACT_APP_ALAN_AI=
\ No newline at end of file
+REACT_APP_ALAN_AI=
+REACT_APP_WIDGETAPIKEY=
```

Comment:
Is there a specific reason to use a new REACT_APP_WIDGETAPIKEY instead of using the existing REACT_APP_APIKEY?

Reply:
LGTM.